arkouda/pdarrayIO.py

```diff
@@ -581,3 +588,146 @@ def save_all(columns : Union[Mapping[str,pdarray],List[pdarray]], prefix_path :
+    return_obj: bool, optional
+        Default False. When True return the Arkouda DataFrame object
+    index: bool, optional
+        Default False. When True, maintain the indexes loaded from the pandas file
+
+    Raises
+    ______
+    RuntimeWarning
+        - Export attempted on Parquet file. Arkouda formatted Parquet files are readable by pandas.
+    RuntimeError
+        - Unsupported file type
+
+    Returns
+    _______
+    pd.DataFrame
+        When `return_obj=True`
+
+    See Also
+    ________
+    pandas.DataFrame.to_parquet, pandas.DataFrame.to_hdf,
+    pandas.DataFrame.read_parquet, pandas.DataFrame.read_hdf,
+    ak.export
+
+    Notes
+    _____
+    - Import can only be performed from hdf5 or parquet files.
+    - Import is only designed to import data from Pandas. Additional format support will require attribute validation
+    to occur before parsing the file.
+    """
+    from arkouda.dataframe import DataFrame
+    # verify file path
+    is_glob = False if os.path.isfile(read_path) else True
+    print(is_glob)
+    file_list = glob.glob(read_path)
+    if len(file_list) == 0:
+        raise FileNotFoundError(f"Invalid read_path, {read_path}. No files found.")
+
+    # access the file type - multiple files valid here because parquet supports glob. Check first listed.
+    file = read_path if not is_glob else glob.glob(read_path)[0]
+    filetype = get_filetype(file)
+    # Note - in the future if we support more than pandas here, we should verify attributes.
+    if filetype == "HDF5":
+        if is_glob:
+            raise RuntimeError("Pandas HDF5 import supports valid file path only. "
+                               "Only supports the local file system, "
+                               "remote URLs and file-like objects are not supported.")
+        df_def = pd.read_hdf(read_path)
+    elif filetype == "Parquet":
+        # parquet supports glob input in pandas
+        df_def = pd.read_parquet(read_path)
+    else:
+        raise RuntimeError("File type not supported. Import is only supported for HDF5 and Parquet file formats.")
+    df = DataFrame(df_def)
+
+    if write_file:
+        df.save_table(write_file, index=index, file_format=filetype)
+
+    if return_obj:
+        return df
+
+
+def export(read_path: str, dataset_name: str = "ak_data", write_file: str = None,
+           return_obj: bool = False, index: bool = False):
+    """
+    Export data from arkouda to Pandas readable format
+
+    Parameters
+    __________
+    rd_filename: str
+        path to file where pandas data is stored.
+    dataset_name: str
+        name to store dataset under
+    index: bool
+        Default False. When True, maintain the indexes loaded from the pandas file
+    write_file: str, optional
+        path to file to write arkouda formatted data to. Only write the file if this is set
+    return_obj: bool, optional
+        Default False. When True return the Arkouda DataFrame object
```

Comment:
Intuitively, I feel like the default should be True.

"When True **return the Arkouda DataFrame object**": This seems to contradict line 694, which says a pd.DataFrame object will be returned.

Reply:
Again, missed fixing the docstring. Initially I had this always writing the file, but because that is also optional, it is probably best to default to returning the object. I updated both import and export to return the object and corrected the docstring wording.